_includes/head.html

```diff
@@ -31,12 +30,12 @@
   <link href="{{ 'feed.xml' | absolute_url }}" type="application/rss+xml" rel="alternate" title="{{ site.name }} Last 10 blog posts" />
 
   {% if site.local_fonts %}
-    {% stylesheet fonts %}
+    {% include stylesheet.html path="fonts" %}
   {% endif %}
 
   {% if site.blog_theme == "light" %}
-    {% stylesheet light %}
+    {% include stylesheet.html path="light" %}
```

Comment:
Only reason is that jekyll-assets outputs unnecessary tags for html5 `</script>` and `<link>`:

`<link type="text/css" rel="stylesheet" href="/assets/light.css">`
,should be:
`<link rel="stylesheet" href="/assets/light.css">`

and

`<script type="text/javascript" src="/assets/vendor.js"></script>`
,should be:
`<script src="/assets/vendor.js"></script>`